downloads.py

```diff
@@ -18,38 +20,84 @@ def is_version_packed(fname):
     return resp.status_code == 200
 
 
-def pack_version(fname, ver_obj, pmgr_obj):
+def pack_version(fname, ver_obj, pmgr_obj, installer_cls):
     if not ver_obj['downloadUrl']:
         return
 
     with zipfile.ZipFile(fname, 'w', zipfile.ZIP_DEFLATED) as ziph:
-        # pack main file
-        download_into_zip(ziph, ver_obj['downloadUrl'], os.path.join("lib", "ext"))
+        if installer_cls is not None:
```

Comment:
Why this if is so high? Let's have single flow, only varying with call to installer as one of the steps...